web/modules/custom/helloworld/helloworld.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'HelloWorld'
+type: module
+description: 'A Custom Module for the Testing Purpose.'
+package: Custom
+core_version_requirement: ^9 || ^10
+dependencies:
+  - drupal:block
+  - drupal:views
+  - mymodule:mymodule
+configure: helloworld.admin_settings
```

Comment:
This red marker at the end means that there is no newline at the end of file.
Please configure VS code with proper plugins so that newline is automatically added to the end of the file.

Please fix this for all the files.

Reply:
Ok i am fixing them